src/esp/metadata/managers/PhysicsAttributesManager.h

```diff
@@ -37,10 +37,10 @@ class PhysicsAttributesManager
    * from the specified configuration file.
    *
    * If a template exists with this handle, this existing template will be
-   * overwritten with the newly created one if @ref registerTemplate is true.
+   * overwritten with the newly created one if registerTemplate is true.
    *
    * @param physicsFilename The configuration file to parse. Defaults to the
-   * file location @ref ESP_DEFAULT_PHYS_SCENE_CONFIG_REL_PATH set by cmake.
+   * file location ESP_DEFAULT_PHYS_SCENE_CONFIG_REL_PATH set by cmake.
```

Comment:
I think this should still be `@ref` :thinking: 

If doxygen complains it can't find the target, it's probably because the variable is not documented anywhere. Alternatively I'd do at least this to make it look like a code thing:

```suggestion
   * file location `ESP_DEFAULT_PHYS_SCENE_CONFIG_REL_PATH` set by cmake.
```

Reply:
It's constructed by cmake and ends up living in esp/physics/configure.h